pkg/aws/actuator/actuator.go

```diff
@@ -880,6 +967,40 @@ func (a *AWSActuator) getDesiredUserPolicy(entries []minterv1.StatementEntry, us
 	return string(b), nil
 }
 
+func (a *AWSActuator) getCloudCredentialsSecret(ctx context.Context, logger log.FieldLogger) (*corev1.Secret, error) {
+	cloudCredSecret := &corev1.Secret{}
+	if err := a.Client.Get(ctx, types.NamespacedName{Name: rootAWSCredsSecret, Namespace: rootAWSCredsSecretNamespace}, cloudCredSecret); err != nil {
+		msg := "unable to fetch cloud cred secret"
+		logger.WithError(err).Error(msg)
+		return nil, &actuatoriface.ActuatorError{
+			ErrReason: minterv1.CredentialsProvisionFailure,
+			Message:   fmt.Sprintf("%v: %v", msg, err),
+		}
```

Comment:
Ah do we now get a condition if the root creds don't exist? I think I had a todo for this, so that's great if so.

Reply:
yep. any failure to provision should give us at least the CredentialsProvisionFailure condition now